docs/3_guides/9_aligned_cli.md

```diff
@@ -0,0 +1,131 @@
+
+    - Options:
+        - `--verification_key_file <Verification key file path>`: Path to verification data file generated from proof submission with the Aligned CLI.
+        - `--proving_system <Proving system>`: Proof system of the supplied verification data file.
+            - **possible values**: `GnarkPlonkBls12_381`, `GnarkPlonkBn254`, `Groth16Bn254`, `SP1`, `Risc0`
+        - `--output <Output file>`: File path for output to be written to.
+
+- `deposit-to-batcher [OPTIONS] --keystore_path <Path to local keystore> --amount <Amount to deposit>`: Deposits Ethereum into the Aligned Layer's `BatcherPaymentService.sol` contract to pay for users proof submission.
+
+    - Options:
+        - `--keystore_path <Path to local keystore>`: Path to the local keystore of the user's wallet. 
+        - `--rpc-url <Ethereum RPC provider address>`: User's Ethereum RPC Node URL. 
+        - `--network <The working network's name>`: The Ethereum network of the Aligned Layer Network the user is querying their balance on.
+            - **default**: `devnet`
+            - **possible values**: `devnet`, `holesky`, `holesky-stage`, `mainnet`
+        - `--amount <Amount to deposit>`: Amount of Ethereum in Wei to deposit into the `BatcherPaymentService.sol` contract to pay for proof submission.
+
+- `get-user-balance [OPTIONS] --user_addr <The user's Ethereum address>`: Retrieves the user's current balance in Aligned Layer's `BatcherPaymentService.sol` contract.
+
+    - Options:
+        - `--network <The working network's name>`: The Ethereum network of the Aligned Layer Network the user is querying their balance on.
+            - **default**: `devnet`
+            - **possible values**: `devnet`, `holesky`, `holesky-stage`, `mainnet`
+        - `--rpc_url <Ethereum RPC provider address>`: User's Ethereum RPC Node URL.
+            - **default**: `http://localhost:8545`
+        - `--user_addr <The user's Ethereum address>`: User's Ethereum address on the provided network. 
+
+- `get-user-nonce [OPTIONS] --user_addr <The user's Ethereum address>`:  Retrieves the user's current nonce from the batcher.
+
+    - Options:
+        - `--batcher_url <Batcher connection address>`: Websocket URL for the Aligned Layer batcher.  
+            - **default**: `ws://localhost:8080` i.e. `devnet`
+        - `--user_addr <The user's Ethereum address>`: User's Ethereum address on Ethereum Mainnet.
+
+## Example:
+
+An example workflow for using the Aligned CLI (Runnable from the root directory of the Aligned Layer Repository):
+
+1. Deposit funds to the batcher.
+```bash
+aligned deposit-to-batcher --network holesky --amount 0.5ether --keystore_path <KEYSTORE_PATH>
```

Comment:
deposit to batcher spec is in wei and it is using ether. Check which one is correct